src/libs/ajax.js

```diff
@@ -1110,9 +1110,23 @@ const DataRepo = signal => ({
       details: async () => {
         const res = await fetchDataRepo(`repository/v1/snapshots/${snapshotId}`, _.merge(authOpts(), { signal }))
         return res.json()
+      },
+      exportSnapshot: async () => {
+        const res = await fetchDataRepo(`repository/v1/snapshots/${snapshotId}/export`, _.merge(authOpts(), { signal }))
```

Comment:
Worth noting: there are other URL parameters that can be passed in to export:
- `exportGsPaths`: can be ignored for now and defaults to false
- `validatePrimaryKeyUniqueness`: this is set to true for now but if we're working with azure snapshots, that needs to be set to false (for now) or the export job will fail.  The cloud provider for the snapshot is returned as part of the snapshot now (as of yesterday)
Does it make sense to make that change now or as a followon?

Reply:
I think followon, if at all. I think there's conversations we need to have internally to the catalog on whether we want to support what here, or just use sensible defaults and obfuscate it from the non-power users.

Reply:
Right.  I do think we'll want the ability to set it in the call (and not expose it to user...but just to make sure it works from Catalog). By setting that, we'll be able to support azure snapshots starting today (which is super cool)

Reply:
Lets do it as a follow on, because we haven't yet trial ran azure snapshots in the catalog, so we may need to make some metadata changes to be able to detect it. @tcjiang9 this may relate to schema definitions.

Reply:
ok sounds good!